### PR TITLE
fix #71639 gumtree.com.au

### LINKED ISSUE #71639: gumtree.com.au
### Issue URL (Incorrect Blocking)
 [https://www.gumtree.com.au/m-my-ads.html?c=1](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.gumtree.com.au%2Fm-my-ads.html%3Fc%3D1)
### Comment
Username: @edmundguan
> can not access the website as formal.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>EasyList China, <br/>Thai Ads Filters

### PR DESCRIPTION
**Pull request by stanislav-atr**
#71639

### CODE REVIEW

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -491,6 +491,8 @@ lmgtfy.app#@##bottom_ads
 4snip.pw#@##DividerAd
 ! neads.io is broken
 @@||neads.io^$domain=neads.io
+! https://github.com/AdguardTeam/AdguardFilters/issues/71639
+@@||static.gumtree.com.au^$domain=gumtree.com.au
```
Comment (zloyden) [inline EnglishFilter/sections/whitelist.txt:495]: Wrong filter. It's caused by Liste FR. Add the exception to the AdGuard French.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/whitelist.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/whitelist.txt:495]: Btw, is it really necessary to whitelist this domain? Something like that `@@/myads.$domain=gumtree.com.au` is enough.

Comment (stanislav-atr) [reply EnglishFilter/sections/whitelist.txt:495]: @zloyden, maybe `/myads.$domain=~gumtree.com.au` then?

Comment (zloyden) [reply EnglishFilter/sections/whitelist.txt:495]: It's better to add the exception here because of `/myads.$domain=~myads.agency`.

**Review by zloyden — APPROVED** [review]

**Pull request merged**